### PR TITLE
De-ES6fy

### PR DESCRIPTION
**Pull request by tricknotes**
We want to remove extra gem dependency.

### CODE REVIEW

**Comment by tricknotes** [comment]
I checked on my local with daimon-news.
Could someone check this PR too?

**Comment by okkez** [comment]
Looks nice :crown:

File: app/assets/javascripts/external_component.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    var $el = $(e);
+
+    $.ajax({
+      type: 'GET',
+      url: $el.data('external-src'),
+      xhrFields: {
+        withCredentials: true
+      }
+    }).then(function(html) {
+      $el.replaceWith(html)
```
Comment (tricknotes) [inline app/assets/javascripts/external_component.js]: Ouch, I missed to add tailing semicolon.

Comment (tricknotes) [inline app/assets/javascripts/external_component.js]: fixed & force pushed.

File: app/assets/javascripts/external_component.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+      type: 'GET',
+      url: $el.data('external-src'),
+      xhrFields: {
+        withCredentials: true
+      }
+    }).then(function(html) {
+      $el.replaceWith(html);
+    });
+  });
+})
```
Comment (okkez) [inline app/assets/javascripts/external_component.js]: missing `;` ?

Comment (tricknotes) [inline app/assets/javascripts/external_component.js]: Right!

**Comment by mtsmfm** [comment]
LGTM

**Pull request merged**